scale/scheduler/threads/recon.py

```diff
@@ -7,8 +7,8 @@
 from scheduler.threads.base_thread import BaseSchedulerThread
 
 
-THROTTLE = datetime.timedelta(seconds=60)
-WARN_THRESHOLD = datetime.timedelta(seconds=1)
+THROTTLE = datetime.timedelta(seconds=1)
```

Comment:
Did you mean to change this from 60 seconds to 1 second or were you meaning to do 1 minute?